libs/libcommon/src/libcommon/storage_options.py

```diff
@@ -2,18 +2,11 @@
 # Copyright 2023 The HuggingFace Authors.
 from dataclasses import dataclass
 
-from libcommon.s3_client import S3Client
-from libcommon.storage import StrPath
+from libcommon.storage_client import StorageClient
 
 
 @dataclass
-class DirectoryStorageOptions:
+class StorageOptions:
```

Comment:
storage_option is already a concept in `fsspec` (it refers to the kwargs passed to a filesystem)

Maybe we can rename this "PublicAssetsStorage" or something like that ?

Reply:
Thanks! It makes sense, I changed to `PublicAssetsStorage`